src/orders-update.js

```diff
@@ -137,18 +137,16 @@ export default class OrdersUpdate {
         if (total === 1) {
           const existingOrder = existingOrders[0]
           const actions = this.buildUpdateActions(order, existingOrder)
-
-          // Do not call the API when there are no changes
-          if (actions.length > 0)
-            return this.client.orders
+          if (actions.length === 0)
+            // eslint-disable-next-line no-throw-literal
+            throw `No valid action found for ${order.orderNumber}`
```

Comment:
Wouldnt it make more sense to just log this one out instead of throwing? With this implementation, it should stop the import completely as we're running them all here? 
https://github.com/commercetools/orders-update/blob/24-log-is-no-action/src/orders-update.js#L179-L180

Reply:
Then it will silently say that the import is worked and updated the given orders though it is not or are you saying that i have to change here https://github.com/commercetools/orders-update/blob/5d21128929314e2b1cc47e456bc4107b61925935/src/orders-update.js#L52

Reply:
I mean that I think the current implementation is a breaking change since it will stop the import completely. 

For example, if we want to update 10 orders and only 9 of them can actually be changed. Before this PR we would change 9 of them and log out success, without telling the user that the 10th order couldn't be updated, right?

That would mean that with this PR we will throw when the failing order is about to be processed. Which then stops the whole process. Meaning that if it has successfully updated 3 orders, and the fourth one is failing, it will stop all the remaining 6 orders even if they are correct.

Does that make sense?

Reply:
@daern91  Done. I thought only single promise will fail not all. I have updated the comment as per your suggestion.